pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -655,16 +655,13 @@ var _ = Describe("Pod Network", func() {
 				mockNetwork.EXPECT().LinkByName(podnic.podInterfaceName).Return(macvtapInterface, nil)
 			})
 
-			XIt("Should pass a non-privileged macvtap interface to qemu", func() {
-				// TODO the test fails in this commit since the domainCache is not stored anywhere, the next commit is removing the domain cache usgae so it will fix the test
+			It("Should pass a non-privileged macvtap interface to qemu", func() {
 				domain := NewDomainWithMacvtapInterface("default")
 
 				api.NewDefaulter(runtime.GOARCH).SetObjectDefaults_Domain(domain)
 				driver, err := podnic.newLibvirtSpecGenerator(domain)
```

Comment:
would it make sense to rename from driver to the specGenerator (or something alike) ? 

(I'm aware this is a leftover - probably from me).